roles/openshift_cloud_provider/tasks/aws.yml

```diff
@@ -0,0 +1,18 @@
+- name: Set AWS provider facts
+  openshift_facts:
+    role: aws
+    local_facts:
+      access_key: "{{ openshift_cloud_provider_aws_access_key | default(None) }}"
+      secret_key: "{{ openshift_cloud_provider_aws_secret_key | default(None) }}"
```

Comment:
Instead of using local facts here, should we instead use openshift_env, and store these under a 'cloud_provider' role (`openshift.cloud_provider.aws.access_key`)?